packages/emails/email-manager.ts

```diff
@@ -56,7 +58,57 @@ const sendEmail = (prepare: () => BaseEmail) => {
   });
 };
 
-const eventTypeDisableAttendeeEmail = (metadata?: EventTypeMetadata) => {
+export const fetchOrganizationEmailSettings = async (organizationId?: number | null | undefined) => {
+  if (!organizationId) return null;
+  const repo = new OrganizationSettingsRepository(prisma);
+  return await repo.getEmailSettings(organizationId);
+};
+
+export const shouldSkipAttendeeEmailWithSettings = (
+  metadata: EventTypeMetadata | undefined,
+  organizationSettings: Awaited<ReturnType<typeof fetchOrganizationEmailSettings>>,
+  emailType?:
+    | "confirmation"
+    | "cancellation"
+    | "rescheduled"
+    | "request"
+    | "reassigned"
+    | "awaiting_payment"
+    | "reschedule_request"
+    | "location_change"
+    | "new_event"
```

Comment:
An ENUM type here would be better and scalable

Reply:
@CarinaWolli @volnei you guys think it (Emails Enum type) is worth doing that at DB level or no?